frontend/src/pages/AppDetails/AppInfo.tsx

```diff
@@ -51,7 +52,12 @@ class AppInfo extends React.Component<AppInfoProps, AppInfoState> {
     if (!this.props.app) {
       return;
     }
-    this.graphDataSource.fetchForVersionedApp(this.props.duration, this.props.app.namespace.name, this.props.app.name);
+    this.graphDataSource.fetchForVersionedApp(
+      this.props.duration,
+      this.props.app.namespace.name,
+      this.props.app.name,
+      this.props.cluster
```

Comment:
So it should change here: 

`this.props.app.cluster ? this.props.app.cluster : ''`